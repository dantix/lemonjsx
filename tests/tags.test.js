import h from '../src/index';

describe('tags', () => {
  describe('attributes', () => {
    it('should omit unsupported attributes', () => {
      expect(<section wrong="value" />).toMatchSnapshot();
    });

    it('should omit unsupported attributes', () => {
      expect(<bar background="value" />).toMatchSnapshot();
    });
  });
});
