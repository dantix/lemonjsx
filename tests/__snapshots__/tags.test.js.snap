exports[`tags attributes should omit unsupported attributes 1`] = `""`;

exports[`tags attributes should omit unsupported attributes 2`] = `
"
"
`;

exports[`tags general should omit unknown tags 1`] = `""`;
